docs/outputs.md

```diff
@@ -0,0 +1,27 @@
+# Outputs
+
+Outputs from Parthenon are controled via ```<parthenon/output*>``` blocks, where ```*``` should be replaced by a unique integer for each block.
+
+## HDF5
+
+Parthenon allows users to select which fields are captured in the HDF5 (```.phdf```) dumps at runtime.  In the input file, include a ```<parthenon/output*>``` block, list of variables, and specify ```file_type = hdf5```.  A ```dt``` parameter controls the frequency of outputs for simulations involving evolution. A ```<parthenon/output*>``` block might look like
+```
+<parthenon/output1>
+file_type = hdf5
+variables = density, velocity, & # comments are still ok
+            energy               # notice the & continuation character
+                                 # for multiline lists
+dt = 1.0
+```
+
```

Comment:
> Maybe add that this will produce an hdf5 (.phdf) output file every 1 units of simulation time containing the density, velocity, and energy of each cell.

Will do.

> So I can add multiple output blocks with type hdf5 and each would result in a different series of files?

Don't know. I didn't write this bit. @jdolence ?